### PR TITLE
add mock aws service for development

### PR DESCRIPTION
**Pull request by dsifford**
This PR adds a fully mocked aws s3 service to use in development.

Let me know if you have any questions.

### CODE REVIEW

File: docker-compose.override.yml [hunk 1]
```diff
@@ -4,13 +4,23 @@
 version: '3.6'
 
 services:
+  # Locally mocked aws services
+  aws:
+    image: localstack/localstack
+    ports:
+      - 4572:4572
+      - 8081:8080
```
Comment (dsifford) [inline docker-compose.override.yml:12]: **Note:** While running in development, navigating to `localhost:8081` will bring up a web UI showing the aws services visually.

Not super useful, but it's there.

Comment (sloan-seamus) [reply docker-compose.override.yml:12]: Is 4572 the port S3 is being run on?

**Review by sloan-seamus — APPROVED** [review]

File: docker-compose.override.yml (hunk 1 above)
Comment (dsifford) [reply docker-compose.override.yml:12]: Yes

**Review by ej3076 — APPROVED** [review]
🏅

**Pull request merged**